packages/webview_flutter/webview_flutter_android/pigeons/android_webview.dart

```diff
@@ -4,18 +4,40 @@
 
 import 'package:pigeon/pigeon.dart';
 
+@ConfigurePigeon(
+  PigeonOptions(
+    dartOut: 'lib/src/android_webview.pigeon.dart',
+    dartTestOut: 'test/test_android_webview.pigeon.dart',
+    dartOptions: DartOptions(copyrightHeader: <String>[
+      'Copyright 2013 The Flutter Authors. All rights reserved.',
+      'Use of this source code is governed by a BSD-style license that can be',
+      'found in the LICENSE file.',
+    ]),
+    javaOut:
+        'android/src/main/java/io/flutter/plugins/webviewflutter/GeneratedAndroidWebView.java',
+    javaOptions: JavaOptions(
+      package: 'io.flutter.plugins.webviewflutter',
+      className: 'GeneratedAndroidWebView',
+      copyrightHeader: <String>[
+        'Copyright 2013 The Flutter Authors. All rights reserved.',
+        'Use of this source code is governed by a BSD-style license that can be',
+        'found in the LICENSE file.',
+      ],
+    ),
+  ),
+)
 class WebResourceRequestData {
-  String? url;
-  bool? isForMainFrame;
-  bool? isRedirect;
-  bool? hasGesture;
-  String? method;
-  Map<String?, String?>? requestHeaders;
+  late String url;
+  late bool isForMainFrame;
+  late bool isRedirect;
+  late bool hasGesture;
+  late String method;
+  late Map<String?, String?> requestHeaders;
```

Comment:
Can't we do a body-less constructor using `this.` references for the arguments, instead of everything being late?

Reply:
Yea both formats are supported. I chose this one because it was less redundant (except for the `late` keyword). I'm fine with either, though.